index.html

```diff
@@ -500,12 +528,17 @@ <h4 class="contact_head">Mubaris NK</h4>
 
 
 
+
 </body>
 <script src="https://unpkg.com/vue"></script>
 <script src="js/mentor.js"></script>
 <script src="js/contestant.js"></script>
 <script src="js/projects.js"></script>
+<<<<<<< HEAD
```

Comment:
resolve theses conflict lines, i had to revert the PR after merging because of this.Make a new PR